internal/backend/windows/wslexe_windows.go

```diff
@@ -89,3 +90,13 @@ func (Backend) State(distributionName string) (s state.State, err error) {
 
 	return state.NotRegistered, nil
 }
+
+// Install installs a new distro from the Windows store.
+func (b Backend) Install(ctx context.Context, appxName string) (err error) {
+	cmd := exec.CommandContext(ctx, "wsl.exe", "--install", appxName, "--no-launch")
```

Comment:
I think you can add the comment about the selection of --no-launch in the code too.